index.bs

```diff
@@ -1925,15 +1925,19 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
     [=authenticators=] can be <a href="https://en.wikipedia.org/w/index.php?title=Hot_plug">hot-plugged</a> into (e.g., via USB)
     or discovered (e.g., via NFC or Bluetooth) by the [=client=] by various mechanisms, or permanently built into the [=client=].
 
-1. If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is present with the value
+1. If <code>|options|.{{CredentialCreationOptions/mediation}}</code> is present with the value
      {{CredentialMediationRequirement/conditional}}:
 
-    1. If |conditionalCreateLifetimeTimer| is expired or |conditionalCreateOrigin| is not |callerOrigin|, throw a "{{NotAllowedError}}" {{DOMException}}.
+    1. If the user agent has not recently mediated an authentication, the origin of said authentication is not |callerOrigin|, or the user
+    does not consent to this type of credential creation, throw a "{{NotAllowedError}}" {{DOMException}}.
 
-    Note: |conditionalCreateLifetimeTimer| and |conditionalCreateOrigin| will be set by the user agent after it believes an authentication ceremony has
-    been completed and the user consents to this type of credential creation.
+    Note: The user agent will note when it believes an authentication ceremony has
+    been completed.
```

Comment:
This needs to be indented one step further in order to not break up the numbered list.

I would also not make this a "Note" since those are [supposed to be non-normative](https://github.com/w3c/webauthn/issues/1979), and this is a fairly important piece.

```suggestion
        It is up to the user agent to decide when it believes an authentication ceremony has been completed.
        That authentication ceremony MAY be performed using other means than the [=Web Authentication API=].
```